src/btree/bt_delete.c

```diff
@@ -194,11 +194,21 @@ __wti_delete_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
      */
     WT_ERR(__wt_page_parent_modify_set(session, ref, false));
 
-    WT_ERR(__wt_calloc_one(session, &ref->page_del));
-
-    /* History store truncation is non-transactional. */
-    if (!WT_IS_HS(session->dhandle))
+    /*
+     * Allocate and initialize `page_del` for pages, excluding those in the history store.
+     *
+     * A `null` `page_del` means that a fast-truncated page is globally visible. Since truncation in
```

Comment:
np: I don't like the "`", is it used for linting reasons?

Reply:
It is simply to indicate that it's a variable name, so people don't confuse it with the text. 
When they look at `page_del` it is much clearer that it is a var name